spec/fixtures/deployment.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
This file was pulled from the https://docs.gitlab.com/ce/api/deployments.html#list-project-deployments documentation